src/components/extensive/Modal/ModalApprove.tsx

```diff
@@ -69,35 +108,47 @@ const ModalApprove: FC<ModalApproveProps> = ({
         <div className="mb-6 flex flex-col rounded-lg border border-grey-750">
           <header className="flex items-center border-b border-grey-750 bg-neutral-150 px-4 py-2">
             <Text variant="text-12" className="flex-[2]">
-              {t("Name")}
+              {"Name"}
             </Text>
             <Text variant="text-12" className="flex flex-1 items-center justify-start">
-              {t("Polygon Check")}
+              {"Polygon Check"}
             </Text>
             <Text variant="text-12" className="flex flex-1 items-center justify-center">
-              {t("Approve")}
+              {"Approve"}
```

Comment:
Please bring back these translations.

Reply:
this component does not require translations, to make this clearer I just moved to the admin folder